### DISCUSSION TITLE
alternative to right clicking to launch Infinite generations

### DISCUSSION THREAD

**Discussion by patrickchamp** [lllyasviel/Fooocus#2468] [Ideas]
As an iPad user connecting remotely via browser, I don’t have the option of right clicking the generate button to select infinite generations. 

if there was a way to do this via checkbox, It would be great though I get that this is a niche use case.

**Comment by mashb1t** [comment]
at least from personal experience and on Android it's possible to do it using the Chrome mobile app and long press the generate button. Also switching to desktop mode is possible in Chrome.
I assume you're using Safari, right?

**Reply by mashb1t** [reply]
There also seems to be a compatibility / accessibility setting for right click, please check this out (EDIT most likely only for use with accessories):
https://www.idownloadblog.com/2020/06/08/ipad-right-click-tutorial/

**Reply by patrickchamp** [reply]
Thanks for doing some digging. I tried Edge safari and chrome but on apple mobile devices they all use the same webkit(?) UI element and behave the same. 

I did some additional digging and it looks like you’re right that right clicking is only possible with a mouse or other device. There’s no way to simulate it via touch with accessibility options. 

This isn’t a big need tbh since i’m able to generate many images by increasing the Max batch count in the preset config. I just figured other people might have the same issue so i figured i’d suggest it.
